Integrations/integration-fireeye.yml

```diff
@@ -54,7 +54,7 @@ script:
             url: 'reports/report',
             method: 'GET',
         },
-        'fe-submit-status': {
+        'fe-submit-file-status': {
```

Comment:
@sgosai I think it might be better to keep this as before for backwards compatibility

Reply:
@hirshbergshachar, integration has been updated.